partner_ref_unique/tests/test_res_partner_ref.py

```diff
@@ -0,0 +1,107 @@
+# Copyright 2017 Tecnativa - Vicent Cubells
+# Copyright 2020 Tecnativa - João Marques
+# Copyright 2020 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.exceptions import ValidationError
+from odoo.tests import common
+
+
+class TestResPartnerRefUnique(common.SavepointCase):
```

Comment:
```suggestion
class TestResPartnerRefUnique(common.TransactionCase):
```
SavepointCase is a deprecated class

Reply:
Updated in latest commit